include/dsd.h

```diff
@@ -59,7 +59,9 @@
 #include "p25p1_heuristics.h"
 
 //OSS support
+#if defined(__linux__)
```

Comment:
```suggestion
#if !defined(__APPLE__) && !defined(__MACH__)
```

Not sure, but might be better to use 

#if !defined(__APPLE__) && !defined(__MACH__)

for this, least that's what was done with m17-fme and apple.

For reference:

https://github.com/lwvmobile/m17-fme/pull/2